platform/linux-generic/test/example/l2fwd_simple/pktio_env

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+#
+# Copyright (C) 2020 Marvell.
```

Comment:
For consistency please use this style for all copyright lines:
`Copyright (c) 2020, Marvell`